templates/islandora-paged-tei-seadragon-viewer.tpl.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @file
+ * Template file for the paged TEI seadragon viewer.
+ */
+?>
+<?php if ($tei_pages_populated): ?>
```

Comment:
Should be `$pages_populated` perhaps a global find and replace gone wrong.
https://github.com/discoverygarden/islandora_paged_tei_seadragon/blob/7.x/theme/islandora-paged-tei-seadragon-viewer.tpl.php#L7